packages/google_maps_flutter/lib/src/convert.dart

```diff
@@ -4,10 +4,6 @@
 
 part of google_maps_flutter_tizen;
```

Comment:
How do you think of this instruction?

> Note: You may have heard of the `part` directive, which allows you to split a library into multiple Dart files. We recommend that you avoid using `part` and create mini libraries instead.

https://dart.dev/guides/libraries/create-library-packages#organizing-a-library-package

The part directive was used in webview_flutter_tizen because we wanted to keep the exact copy of the original code which contained private types/functions.

Reply:
IMO, It is currently under development, so this may be applied when it is merged into the master.
Of course, it will be convenient in the future if it is organized from now on.

Reply:
This plugin is being implemented similar to original plugin structure. Of course, It could be changed in the form of a mini-library, but I think it's probably after the first release.

Reply:
Okay, I see.